code/game/objects/items/ego_weapons/waw.dm

```diff
@@ -24,18 +24,38 @@
 	desc = "A sword suitable for swift thrusts. \
 	Even someone unskilled in dueling can rapidly puncture an enemy using this E.G.O with remarkable agility."
 	icon_state = "despair"
-	force = 30
+	force = 20
 	damtype = WHITE_DAMAGE
 	armortype = WHITE_DAMAGE
 	w_class = WEIGHT_CLASS_NORMAL
 	attack_verb_continuous = list("stabs", "attacks", "slashes")
 	attack_verb_simple = list("stab", "attack", "slash")
 	hitsound = 'sound/weapons/ego/rapier1.ogg'
+	var/combo = 0
 	attribute_requirements = list(
 							JUSTICE_ATTRIBUTE = 80
 							)
 
+//This is like an anime character attacking like 4 times with the 4th one as a finisher attack.
 /obj/item/ego_weapon/despair/melee_attack_chain(mob/user, atom/target, params)
 	..()
-	user.changeNext_move(CLICK_CD_MELEE * 0.75)
-	hitsound = "sound/weapons/ego/rapier[pick(1,2)].ogg"
+	if(combo < 4)
+		force = 20
+		user.changeNext_move(CLICK_CD_MELEE * 0.5)
+		hitsound = "sound/weapons/ego/rapier[pick(1,2)].ogg"
```

Comment:
Why put it everywhere if the result is always the same?
Also see how Justitia does it, you should've put it in the other proc.

Reply:
Fixed